test/test_torch.py

```diff
@@ -1695,7 +1695,18 @@ def test_add(self):
         m2 = torch.tensor([True, True, False, False, False, True], dtype=torch.bool)
         self.assertEqual(m1 + m2, torch.tensor([True, True, False, True, False, True], dtype=torch.bool))
 
-        # [res] torch.add([res,] tensor1, value, tensor2)
+        # bfloat16
+        m1 = torch.tensor([1, 2], dtype=torch.bfloat16)
```

Comment:
shouldn't the constituents also be `1., 2.` ?  You have the result being that, but not the elements that are actually being added.  Also, how about testing actual decimal values?